Servers/swagger.yaml

```
@@ -4884,3 +5134,30 @@ components:
         orderNo:
           type: integer
           example: 1
+
+    TrainingModule:
+      properties:
+        id:
+         type: integer
```

YAMLlint (1.37.1)

[warning] wrong indentation: expected 10 but found 9

(indentation)